### PR TITLE
Create Standalone Module

### PR DESCRIPTION
**Pull request by ghalestrilo**
## Contribution

This PR adds code for running `mgeval` as a standalone module, including:
  - A `__main__.py` file
  - An update to the documentation, indicating how to use the script
  - A bugfix for `bar_used_pitch` calculation

## Context

I'm developing a system to evaluate the output of different music generationg RNNs, and I'm using `mgeval` to evaluate these models. The system uses python 3 and therefore it's incompatible with `mgeval` as a library. The workaround I found is to run `mgeval` as a standalone module, producing the metrics and saving them to a pickle.

I think this could be an useful feature for other people as well, and I'm willing to make any required adjustments

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+data/
+start.sh
```
Comment (RichardYang40148) [inline .gitignore]:
```suggestion
start.sh
*.pyc
```

**Review by RichardYang40148 — CHANGES_REQUESTED** [review]
Appreciate the contribution!
Everything looks good.
There a few `.pyc` files that are pushed along with the codes, a little touch up in the gitignore will do.

**Comment by ghalestrilo** [comment]
Great! I removed the pycache files, should be good now

**Review by RichardYang40148 — APPROVED** [review]
Thanks!

**Pull request merged**